cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -2034,7 +2034,7 @@ initCommand = CommandUI {
       ++ "which will let you specify the options via flags and will use the defaults for the rest.\n",
     commandNotes = Nothing,
     commandUsage = \pname ->
-         "Usage: " ++ pname ++ " init [FLAGS]\n",
+         "Usage: " ++ pname ++ " init [PROJECT ROOT] [FLAGS]\n",
```

Comment:
I'd change the above text from "Calling init with no arguments" to "Calling init with no flags", since we can now pass a non-flag argument. I think we should also add a sentence describing what [PROJECT ROOT] does. something like "Passing a project root will init in that directory instead of the current directory."